ICM: Edit System does not start because of SAXParseException

Error during server startup:
com.intershop.beehive.core.capi.common.SystemException: Error parsing: /intershop/replication-conf/replication-clusters.xml
at com.intershop.beehive.core.internal.staging.ReplicationClusterConfiguration.init(ReplicationClusterConfiguration.java:374)
at com.intershop.beehive.core.internal.staging.TargetSystemMgrImpl.onReadyHook(TargetSystemMgrImpl.java:60)
at com.intershop.beehive.core.internal.environment.ServerEnvironmentImpl.onReadyHook(ServerEnvironmentImpl.java:887)
at com.intershop.beehive.core.internal.environment.ServerEnvironmentImpl.startup(ServerEnvironmentImpl.java:297)
at com.intershop.beehive.core.internal.environment.ApplicationContextListener.startServerEnvironment(ApplicationContextListener.java:53)
at com.intershop.beehive.core.internal.environment.ApplicationContextListener.lambda$contextInitialized$0(ApplicationContextListener.java:25)
at java.base/java.lang.Thread.run(Unknown Source)
Caused by: jakarta.xml.bind.UnmarshalException
- with linked exception:
[org.xml.sax.SAXParseException; lineNumber: 18; columnNumber: 30; Content is not allowed in trailing section.]
at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.createUnmarshalException(AbstractUnmarshallerImpl.java:290)
at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.createUnmarshalException(UnmarshallerImpl.java:539)
at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal0(UnmarshallerImpl.java:224)
at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal(UnmarshallerImpl.java:189)
at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:146)
at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:188)
at jakarta.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:172)
at com.intershop.beehive.core.internal.staging.ReplicationClusterConfiguration.init(ReplicationClusterConfiguration.java:302)
... 6 more
Caused by: org.xml.sax.SAXParseException; lineNumber: 18; columnNumber: 30; Content is not allowed in trailing section.
at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source)
at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(Unknown Source)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(Unknown Source)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(Unknown Source)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(Unknown Source)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl$TrailingMiscDriver.next(Unknown Source)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(Unknown Source)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source)
at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(Unknown Source)
at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
at org.glassfish.jaxb.runtime.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal0(UnmarshallerImpl.java:218)
... 11 more

--- a/charts/icm-as/.bumpversion.cfg
+++ b/charts/icm-as/.bumpversion.cfg
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.3.1
+current_version = 0.3.2
 commit = true
 tag = false
 message = Bump icm-as chart version: {current_version} → {new_version}

--- a/charts/icm-as/Chart.yaml
+++ b/charts/icm-as/Chart.yaml
@@ -14,7 +14,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.3.1
+version: 0.3.2
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.

--- a/charts/icm-as/templates/config-replication-clusters-xml-cm.yaml
+++ b/charts/icm-as/templates/config-replication-clusters-xml-cm.yaml
@@ -28,7 +28,10 @@ data:
             </target-cluster>
         </target-clusters>
     </replication-configuration>
-{{- end }} # else if fails with "INTERSHOP_DATABASETYPE is not a method but has arguments"
+{{- end }} #
+{{- /*
+else if fails with "INTERSHOP_DATABASETYPE is not a method but has arguments"
+*/}}
 {{- if eq .Values.environment.INTERSHOP_DATABASETYPE "oracle" }}
   replication-clusters.xml:  |-
     <?xml version="1.0"?>

--- a/charts/icm-replication/.bumpversion.cfg
+++ b/charts/icm-replication/.bumpversion.cfg
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.1.1
+current_version = 0.1.2
 commit = true
 tag = false
 message = Bump icm-replication chart version: {current_version} → {new_version}

--- a/charts/icm-replication/Chart.yaml
+++ b/charts/icm-replication/Chart.yaml
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "11.0.0"
 description: Intershop Commerce Management - ICM Replication
 name: icm-replication
-version: 0.1.1
+version: 0.1.2

--- a/charts/icm-replication/requirements.yaml
+++ b/charts/icm-replication/requirements.yaml
@@ -1,9 +1,9 @@
 dependencies:
   - name: icm
-    version: 0.3.1
+    version: 0.3.2
     repository: file://../icm
     alias: icm-live
   - name: icm
-    version: 0.3.1
+    version: 0.3.2
     repository: file://../icm
     alias: icm-edit

--- a/charts/icm/.bumpversion.cfg
+++ b/charts/icm/.bumpversion.cfg
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.3.1
+current_version = 0.3.2
 commit = true
 tag = false
 message = Bump icm chart version: {current_version} → {new_version}

--- a/charts/icm/Chart.yaml
+++ b/charts/icm/Chart.yaml
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "11.0.0"
 description: Intershop Commerce Management - ICM
 name: icm
-version: 0.3.1
+version: 0.3.2

--- a/charts/icm/requirements.yaml
+++ b/charts/icm/requirements.yaml
@@ -1,6 +1,6 @@
 dependencies:
   - name: icm-as
-    version: 0.3.1
+    version: 0.3.2
     repository: file://../icm-as
   - name: icm-web
     version: 0.2.1

